client/quantum_serverless/core/job.py

```diff
@@ -490,6 +491,7 @@ def upload(self, program: QiskitPattern):
                             "entrypoint": program.entrypoint,
                             "arguments": json.dumps({}),
                             "dependencies": json.dumps(program.dependencies or []),
+                            "env_vars": json.dumps(program.env_vars or {}),
```

Comment:
One question here: `env_vars` should be set with the `QiskitPattern` or in the `Job` like arguments? @psschwei @IceKhan13

Reply:
Here is my 2 cents.  For a configuration value, its default is in the environment variable with the pattern and the argument of the Job can overwrite it at the execution time.  So I think that the pattern is better have the env_vars.